example/package.json

```diff
@@ -7,9 +7,12 @@
     "test": "jest"
   },
   "dependencies": {
+    "cache": "^2.1.0",
+    "clean": "^4.0.2",
     "react": "16.6.3",
     "react-native": "0.58.4",
-    "react-native-simple-share": "https://github.com/Ansalibrahim/react-native-simple-share#feature/share-implementation"
+    "react-native-simple-share": "https://github.com/Ansalibrahim/react-native-simple-share#feature/share-implementation",
+    "yarn": "^1.13.0"
```

Comment:
I think you made a mistake here and added a `cache`, `clean` and `yarn` package. Could you remove them? Then I can merge it.

Reply:
ok. Will do soon

Reply:
done

Reply:
You also need to run `yarn` in that folder.

Reply:
And update the `yarn.lock` file.

Reply:
I forgot. done.